servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -518,7 +534,7 @@ public boolean equals(Object object) {
   }
 
   private TraversalResult traverseToInode(String[] pathComponents, boolean persist)
```

Comment:
Since this method is called by `getInodeIdByPath` which may be called frequently, to reduce memory footprint, I would like to keep the `persist` parameter so that when we merely want to traverse the tree, the non-persisted Inodes won't be copied into an array.